content/04.methods.md

```diff
@@ -162,20 +162,26 @@ Given the processed `SingleCellExperiment` object and organ-specific reference,
 For each cell, `CellAssign` calculates a probability of assignment to each cell type in the reference.
 The probability matrix and a prediction based on the most probable cell type were added as cell type annotations to the processed `SingleCellExperiment` object.
 
+For `SCimilarity`, the foundation model described in Heimberg et al. [@doi:10.1038/s41586-024-08411-y], containing 7.3 million cells from various normal and diseased tissues was obtained from zenodo (<https://zenodo.org/records/10685499>) and used to annotate cells in all samples.
+The assigned cell type label and the minimum distance of the query cell to a reference cell in the model were added to the processed `SingleCellExperiment` object.
```

Comment:
This phrasing seems off to me, specifically "to a reference cell."

The [tutorial](https://genentech.github.io/scimilarity/notebooks/cell_annotation_tutorial.html#Annotation-QC) says:
> `min_dist`, represents the minimum distance between a cell in the query dataset and all cells in the training set

So, instead of "a reference cell"...eh? Or am I really wrong here?

```suggestion
The assigned cell type label and the minimum distance of the query cell to reference cells of the same label in the model were added to the processed `SingleCellExperiment` object.
```